substrate-archive/Cargo.toml

```
@@ -6,61 +6,45 @@ edition = "2018"
 
 [dependencies]
 # external
+async-trait = "0.1"
+coil = { git = "https://github.com/insipx/coil", package = "coil" }
```

needless `package name`

```suggestion
coil = { git = "https://github.com/insipx/coil" }
```